daft/daft/__init__.pyi

```diff
@@ -870,6 +870,7 @@ def read_parquet_into_pyarrow(
     io_config: IOConfig | None = None,
     multithreaded_io: bool | None = None,
     coerce_int96_timestamp_unit: PyTimeUnit | None = None,
+    coerce_string_to_binary: bool | None = None,
```

Comment:
Maybe `read_string_as_binary`?

Reply:
+1. 

I was trying to think of a better name for this parameter, but was drawing a blank. I like `read_string_as_binary`

Reply:
Just chatted with @samster25 here are some other options:

```
enable_string_encoding=False
string_encoding="utf-8" | None,  # this one is cool, could let us support others in the future potentially
```

Reply:
oooh i really like 'string_encoding="utf-8" | None' It would make it super easy to switch to [StringView](https://arrow.apache.org/docs/cpp/api/datatype.html#_CPPv4N5arrow4Type4type11STRING_VIEWE) or LargeUtf8

Reply:
I like the `string_encoding` option. Have that implemented now.